### ISSUE TITLE
Warning / Tritt auf im MP und auf dem Server

### ISSUE THREAD

**Issue by DJT0BE** [stijnwop/manureSystem#81]
Moin,

folgendes Warning tritt auf im MP auschließlich beim Admin oder aber auf dem Server in der Server Log.

Tritt auch immer wieder auf, so das die Log voll läuft und die FPS einbrechen.

2020-04-11 23:31 Warning (script): 'setJointFrame': Argument 1 has wrong type. Expected: Int. Actual: Nil 
2020-04-11 23:31   ..\..\..\src\base\scripting\lua\LuaScriptSystem.cpp (882): expectedType == Value::VoidType || expectedType == Value::ClassType
2020-04-11 23:31 LUA call stack:
2020-04-11 23:31   dataS/scripts/player/Player.lua (3013) : setJointFrame
2020-04-11 23:31   dataS/scripts/player/Player.lua (1202) : rotateObject
2020-04-11 23:31   dataS/scripts/utils/Utils.lua (381) : oldFunc
2020-04-11 23:31   dataS/scripts/network/NetworkNode.lua (98) : update
2020-04-11 23:31   dataS/scripts/network/Server.lua (58) : updateActiveObjects
2020-04-11 23:31   dataS/scripts/BaseMission.lua (1500) : update
2020-04-11 23:31   dataS/scripts/FSBaseMission.lua (1977) : update
2020-04-11 23:31   dataS/scripts/missions/mission00.lua (469) : update
2020-04-11 23:31   dataS/scripts/main.lua (1896) : update

MfG

**Comment by stijnwop** [comment]
Update to the latest release here. Has been reported and fixed already.

**Comment by DJT0BE** [comment]
This is the latest Version from GitHub

**Comment by stijnwop** [comment]
Mm, and how to reproduce this?

**Comment by DJT0BE** [comment]
At the moment, we are trying to reproduce the warning.

**Comment by stijnwop** [comment]
Any success?

**Comment by 74656b** [comment]
We ran into this exact same error last night on our server. 

1.0.9.0-beta (From release)

We had a few people on the server, only one person was using a hose, dragging it across a yard to stock fert totes/stock hardi 3point sprayer. Manipulated it a few times moving between totes as he filled up. Once done hose was drug back into a shed, where one end fell through the foundation and got stuck, wrestled with it for a while when it finally freed up. That's when I noticed there was a bunch of these lua errors in the dev console.

Not much I know, but hopefully it helps some.
